src/components/PeopleTable/TableHeader.tsx

```diff
@@ -0,0 +1,49 @@
+import { useSearchParams } from 'react-router-dom';
+import cn from 'classnames';
+import { SearchLink } from '../SearchLink';
+
+const sortTypes = ['name', 'sex', 'born', 'died'];
+
+export const TableHeader = () => {
+  const [searchParams] = useSearchParams();
+  const sort = searchParams.get('sort') || '';
+  const order = searchParams.get('order') || '';
+
+  const linkClass = (param: string) =>
+    cn(
+      'fas',
+      { 'fa-sort': sort !== param },
+      { 'fa-sort-up': sort === param && !order },
+      { 'fa-sort-down': sort === param && order === 'desc' },
+    );
+
+  const linkParam = (param: string) => {
+    return {
+      sort: param,
+      order: sort === param ? (order === 'desc' ? null : 'desc') : null,
+      ...(sort === param && order === 'desc'
+        ? { sort: null, order: null }
+        : {}),
+    };
```

Comment:
its not good idea to use ternary operators in ternary operator, this 3 lines is very hard to read, try to make it in another way